cmd/checkmarxExecuteScan.go

```diff
@@ -93,7 +93,7 @@ func loadExistingProject(sys checkmarx.System, initialProjectName, pullRequestNa
 
 func zipWorkspaceFiles(workspace, filterPattern string) *os.File {
 	zipFileName := filepath.Join(workspace, "workspace.zip")
-	patterns := strings.Split(filterPattern, ",")
+	patterns := strings.Split(strings.ReplaceAll(filterPattern, " ", ""), ",")
```

Comment:
Would we have paths containing whitespaces? Suspect we should do split first and then do a lrtrim on all the pattern parts, right?

Reply:
Maybe doing `strings.ReplaceAll(strings.ReplaceAll(filterPattern, ", ", ","), " ,", ",")` is better.

Reply:
Thanks for this proposal, I took this implementation